### ISSUE TITLE
Abnormal biparietal diameter

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11333]
**Preferred term label:**
Abnormal biparietal diameter

**Synonyms**
Abnormal BPD

**Definition (free text, please give PubMed ID)**
Abnormal biparietal diameter (BPD) refers to a measurement of the fetal head width (from one parietal bone to the other) that falls outside the expected range for a given gestational age.

Comment:
todo, explain how BPD is measured and its importance

**Parent term (use hpo.jax.org/app)**
Fetal head anomaly HP:0034245
Abnormality of skull size HP:0000240

**Your nano-attribution (ORCID)**
0000-0002-0736-9199

**Comment by pnrobinson** [comment]
@VarenyaJ Please comment

**Comment by VarenyaJ** [comment]
> [@VarenyaJ](https://github.com/VarenyaJ) Please comment

Would add something like:
**Definition (free text, please give PubMed ID)**
"Abnormal biparietal diameter (BPD) refers to a fetal head width measurement, taken between the outer edge of the proximal parietal bone and the inner edge of the distal parietal bone on an ultrasound image that lies outside the expected range for a given gestational age"

Will try to find Hadlock reference (e.g. PMID: 6979175 or PMID: 6152941)

**Comment by VarenyaJ** [comment]
Tagging @michellejoywang / https://orcid.org/0000-0002-5177-8109

I'm @ https://orcid.org/0009-0000-4429-6024

**Comment by michellejoywang** [comment]
Expected range would be between 10%ile and 90%ile with microcephaly defined as <3%ile or  2 standard deviations below the mean and macrocephaly defined as >97%ile or over 2 standard deviations above the mean

**Comment by VarenyaJ** [comment]
**Definition (free text, please give PubMed ID)**
"Abnormal biparietal diameter (BPD) refers to a fetal head width measurement, taken between the outer edge of the proximal parietal bone and the inner edge of the distal parietal bone on an ultrasound image, that lies outside the expected range for a given gestational age. The expected range typically falls between the 10th and 90th percentiles, with microcephaly defined as a BPD below the 3rd percentile (or more than 2 standard deviations below the mean) and macrocephaly defined as a BPD above the 97th percentile (or more than 2 standard deviations above the mean)."
- [ ] *Hadlock reference (PMID: 6979175, PMID: 6152941)*

**Preferred term label:**
Abnormal Fetal Biparietal Diameter

**Synonyms**
Abnormal BPD

**Parent term (use hpo.jax.org/app)**
Fetal head anomaly HP:0034245
Abnormality of skull size HP:0000240

**Your nano-attribution (ORCID)**
@pnrobinson: 0000-0002-0736-9199
@michellejoywang 0000-0002-5177-8109
@VarenyaJ 0009-0000-4429-6024

**Comment by pnrobinson** [comment]
@michellejoywang @VarenyaJ 
Adding new term: 
- ID: HP:6001417
- label: Abnormal biparietal diameter
- parents: [Fetal head anomaly (HP:0034245)](https://hpo.jax.org/app/browse/term/HP:0034245)
- pmid: PMID:6152941|PMID:6979175
- def: Abnormal biparietal diameter (BPD) refers to a fetal head width measurement, taken between the outer edge of the proximal parietal bone and the inner edge of the distal parietal bone on an ultrasound image, that lies outside the expected range for a given gestational age. The expected range typically falls between the 10th and 90th percentiles, with microcephaly defined as a BPD below the 3rd percentile (or more than 2 standard deviations below the mean) and macrocephaly defined as a BPD above the 97th percentile (or more than 2 standard deviations above the mean).
- comment: 
- orcid: orcid:0000-0002-5177-8109
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11333
- synonyms: Abnormal BPD(NONE)
